praetorian_cli/sdk/entities/seeds.py

```
@@ -135,8 +135,7 @@ def list(self, seed_type=Kind.SEED.value, key_prefix='', pages=100000) -> tuple:
         elif not seed_type:
             seed_type = Node.Label.SEED
         else:
-            error(f'Invalid seed type: {seed_type}')
-            return ([], None)
+            raise ValueError(f'Invalid seed type: {seed_type}')
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)